trio/_core/_multierror.py

```diff
@@ -130,9 +140,9 @@ def push_tb_down(tb, exc, preserved):
 # frame show up in the traceback; otherwise, we leave no trace.)
 @attr.s(frozen=True)
 class MultiErrorCatcher:
-    _handler = attr.ib()
+    handler: Callable[[BaseException], BaseException | None] = attr.ib()
```

Comment:
Is there a reason to make this public?

Reply:
Oh I didn't intend on doing that, I will fix that

Reply:
Changed in [e0f7a9c](https://github.com/python-trio/trio/pull/2742/commits/e0f7a9cac32647c303af28f101600ade0331a45e)